jetty-util/src/main/java/org/eclipse/jetty/util/IncludeExcludeSet.java

```diff
@@ -188,6 +188,28 @@ public Boolean isIncludedAndNotExcluded(P item)
         return null;
     }
 
+    /**
+     * Test Included
+     *
+     * @param item The item to test
+     * @return Boolean.TRUE if item is included
+     */
+    public Boolean isIncluded(P item)
+    {
+        return _includePredicate.test(item);
+    }
+
+    /**
+     * Test Excluded
+     *
+     * @param item The item to test
+     * @return Boolean.TRUE if item is excluded
+     */
+    public Boolean isExcluded(P item)
+    {
+        return _excludePredicate.test(item);
+    }
+
```

Comment:
These are not supposed to be here.
This is a Predicate that follows the "IncludeExcludeSet" logic.
Where Excludes always win over Includes.
Please remove these.